radis/lbl/factory.py

```diff
@@ -745,6 +751,9 @@ def eq_spectrum(
                 "radis_version": get_version(),
             }
         )
+        time_calc = {}
+        time_calc["time"] = self.time.dict_time
+        conditions.update(time_calc)
```

Comment:
In one line and add it to the dictionary above ?

Warning : make sure to make a copy/deepcopy ; else each Spectrum will still be connected to the Factory conditions (check if not done already?).

Reply:
What about setting `self.time.dict_time = {}` after every Spectrum calculation? This way we can keep each Spectrum calculation to itself even when multiple Spectrums are created using the same SF object?